views/admin/admindashboards/layoutGrid/_pageTitle.cfm

```diff
@@ -0,0 +1,45 @@
+<cfscript>
+	param name="args.icon"              type="string" default="";
+	param name="args.title"             type="string" default="";
+	param name="args.subTitle"          type="string" default="";
+	param name="args.pageHeaderButtons" type="string" default="";
+	param name="args.customImg"         type="string" default="";
+
+	icon         = ReFind( "^fa\-", args.icon ) ? args.icon : "fa-#args.icon#";
+	haveButtons  = Len( Trim( args.pageHeaderButtons ) );
+	hasIcon      = Len( Trim( args.icon ) );
+	hasCustomImg = Len( Trim( args.customImg ) );
+
+	if( hasCustomImg ) {
+		customImg   = event.buildLink( assetId=args.customImg, derivative="customHeaderImg75px" );
+		hasIcon = false;
+	}
+</cfscript>
+
+<cfoutput>
+	<div class="page-header<cfif haveButtons> with-buttons</cfif><cfif hasIcon> with-icon</cfif><cfif hasCustomImg> with-image</cfif>">
+		<h1>
+			<cfif hasIcon >
```

Comment:
Extra space on the end of the two if and else if clauses (and the buttons if at the bottom)